### ISSUE TITLE
Feedback to Android from Admin Panel and DB

### ISSUE THREAD

**Issue by sebastiangaertner** [Greenstand/treetracker-android#429]
We should start working on a simple way to send feedback to the app from the DB. The amount of rejected trees should be communicated to the app for example but there is more info that needs to be synced. 
Lets say we spilt it up into three sections. 

1. Local (on device)
- Total Trees
- Uploaded
- to sync

2. Remote (in Greenstand DB): 
- Trees Total:
- Trees rejected: 76
- Trees approved: 

3. If in payout program (approved Entity ID): 
- Total Trees payed out:
- Trees waiting for payout
- Total amount payed out

1 is showing on first login. 
2. gets send back by the DB to the app on the second sync
3. gets displayed once the person is part of a payout scheme

2 and 3 could be maybe done by a cron job filled static file that is synced each time. Could be as tiny as possible since the sql statement behind gathering this information might be slow? TBD

**Comment by arunbakt** [comment]
I like the way where local device data is supplemented with the remove server data about the Planter's tree approval and payment details.

Explicit rejections in addition to approval is a good idea since it removes ambiguity.

**Comment by deepwinter** [comment]
Please see milestone 1.3.  Several pieces that are prerequisites to this functionality are already under way.  We can refine this ticket into implemention items for github tickets.  It's fine to post issues like this, but better might be a doc that we can review in the Android slack channels and then translated into implementation tickets here on github.

**Comment by rgesulfo** [comment]
I agree with @deepwinter. What would be the doc with all these new features? Do we have designs for each of them? Thanks

**Comment by sebastiangaertner** [comment]
Raff your design was getting there 
![greenstand-github-429](https://user-images.githubusercontent.com/3283800/76827190-59f1ec80-682f-11ea-9058-1b7863504da1.png)

**Comment by EngrTaofeek** [comment]
@sebastiangaertner is this issue still required so we can involve the backend team to achieve this.

**Comment by sebastiangaertner** [comment]
Yes its is one of the vital functions. Currently people have no idea what their account related numbers are.
